tests/config/config.TEMPLATE.mysql.inc.php

```diff
@@ -100,3 +100,8 @@
 
 [debug]
 show_stacktrace = On
+
+[queues]
```

Comment:
Don't forget this comment:
https://github.com/pkp/pkp-lib/pull/7049#issuecomment-848828840

Reply:
@asmecher I had a brief conversation with @NateWr. He told me about some script do create the `docs/RELEASE` correctly.

Reply:
Sorry @henriqueramos! That was my misunderstanding. I thought you were talking about the list of issues under each release. I think for configuration parameters you still need to manually add this information.

Alec, should he be creating a new document under docs/release-notes/README-3.4.0 or replacing docs/RELEASE with something new?

Reply:
There is already a `docs/release-notes/README-3.4.0`; I've synced it with `docs/RELEASE`. Please add configuration change documentation to both.